arrow-libs/fx/arrow-fx-coroutines/api/arrow-fx-coroutines.api

```diff
@@ -182,7 +182,6 @@ public final class arrow/fx/coroutines/ParTraverse {
 	public static final fun parSequence (Ljava/lang/Iterable;Lkotlin/coroutines/Continuation;)Ljava/lang/Object;
 	public static final fun parSequence (Ljava/lang/Iterable;Lkotlin/coroutines/CoroutineContext;Lkotlin/coroutines/Continuation;)Ljava/lang/Object;
 	public static synthetic fun parSequence$default (Ljava/lang/Iterable;Lkotlin/coroutines/CoroutineContext;Lkotlin/coroutines/Continuation;ILjava/lang/Object;)Ljava/lang/Object;
-	public static final fun parSequenceEither (Ljava/lang/Iterable;Larrow/typeclasses/Semigroup;Lkotlin/coroutines/Continuation;)Ljava/lang/Object;
```

Comment:
These were incorrectly named inside `ParTraverseValidated.kt`.. Should we deprecate them for `0.13.3`?